index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Ayoub Chahir | Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <ul>
+        <li id="logo"><a href="#" class="logo">My Logo</a></li>
+        <li><img src="./img/sandwich.svg" alt="Menu Button" /></li>
+      </ul>
+    </header>
+    <div class="background-shapes">
+      <img src="./img/Ellipse-3.svg" class="ellipse-3" alt="" /><img
+        src="./img/Ellipse-4.svg"
+        class="ellipse-4"
+        alt=""
+      /><img src="./img/Ellipse-5.svg" class="ellipse-5" alt="" /><img
+        src="./img/Ellipse-8.svg"
+        class="ellipse-8"
+        alt=""
+      /><img src="./img/Ellipse-11.svg" class="ellipse-11" alt="" /><img
+        src="./img/Ellipse-12.svg"
+        class="ellipse-12"
+        alt=""
+      /><img src="./img/Ellipse-13.svg" class="ellipse-13" alt="" /><img
+        src="./img/Ellipse-14.svg"
+        class="ellipse-14"
+        alt=""
+      /><img src="./img/Ellipse-15.svg" class="ellipse-15" alt="" /><img
+        src="./img/Rectangle-4.svg"
+        class="rect-4"
+        alt=""
+      /><img src="./img/Rectangle-5.svg" class="rect-5" alt="" /><img
+        src="./img/Rectangle-6.svg"
+        class="rect-6"
+        alt=""
+      /><img src="./img/Rectangle-7.svg" class="rect-7" alt="" />
+    </div>
+    <section class="headline">
```

Comment:
- [ ] this section should be displayed `flex` according to project requirement, kindly add a `display: flex` property to `headline` class in your CSS file

Reply:
Thank's, this was very helpful